core/common/assets/js/api/modules/command-base.js

```diff
@@ -1,6 +1,8 @@
 import ArgsObject from 'elementor-assets-js/modules/imports/args-object';
 
 export default class CommandBase extends ArgsObject {
+	isCommandBase = true;
```

Comment:
?

Reply:
There many difference instances that extends CommandBase, by instanceOf alone you cannot know his grant parent which is sometimes the CommandBase, so that solution's i would like to hear if there is any better solutions.

Reply:
Please check this commit: https://github.com/elementor/elementor/pull/10418/commits/e5a3e78432d844203be9d457ecdcc5f59c867eb5